README.md

```diff
@@ -60,6 +60,24 @@ Travis CI testing.
 
 For details on setting up Minikube, see these [instructions](docs/minikube.md).
 
+### Using Kubernetes in Docker for Mac
+
+Docker 18.06 for Mac added built-in support for running a single node Kubernetes cluster.
+Using this capability, you can use Helm to deploy to Kubernetes on your laptop without
+needing to install Minikube or an additional layer of virtualization.
+
+Step-by-step instructions on enabling Kubernetes in Docker are
+available as part of the
+[Getting started](https://docs.docker.com/docker-for-mac/#general)
+documentation from Docker.
+
+In a nutshell, open the Docker preferences pane, switch to the
+Kubernetes panel, and check the box to enable Kubernetes.  You will
+want to use the `kubectl` cli that is installed by Docker in
+`/usr/local/bin` and is configured to access the Kubernetes cluster
+running in Docker, so please make sure it is appears in your path
+before any `kubectl` you have installed on your machine.
+
```

Comment:
I had problems is not enougph to use the `kubectl` from `/usr/local/bin` you need to also change the context to `docker-for-desktop` 
```
$ kubectl config get-contexts
$ kubectl config use-context docker-for-desktop
```